packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -425,7 +425,9 @@ const PersonalDetailsForm = observer(() => {
                                                 is_phone_number_editted: account_settings.phone !== values.phone,
                                             })}
                                             onChange={(e: ChangeEvent<HTMLInputElement>) => {
-                                                handleChange(e);
+                                                let phone_number = e.target.value.replace(/[^0-9]/g, '');
+                                                phone_number = phone_number.length === 0 ? '+' : `+${phone_number}`;
```

Comment:
we are checking for length 0 which can be skipped if a user paste a number having move than 0 digits. 

You can try pasting the phone number in the phone number field

Reply:
The check for length 0 is to prefix the '+' in the phone number field 🙏